pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -119,20 +119,22 @@ self: super: {
     '';
   });
 
-  coc-nvim = let
-    version = "0.0.72";
-    index_js = fetchzip {
-        url = "https://github.com/neoclide/coc.nvim/releases/download/v${version}/coc.tar.gz";
-        sha256 = "128wlbnpz4gwpfnmzry5k52d58fyp9nccha314ndfnr9xgd6r52y";
-      };
-  in super.coc-nvim.overrideAttrs(old: {
-    # you still need to enable the node js provider in your nvim config
-    postInstall = ''
-      mkdir -p $out/share/vim-plugins/coc-nvim/build
-      cp ${index_js}/index.js $out/share/vim-plugins/coc-nvim/build/
-    '';
-
-  });
+  # coc.nvim requires a build/index.js file that's only available in the release version. As such,
+  # auto-updating isn't possible with the current script since master doesn't contain this file.
+  # The previous fix for this added the index.js file in for a given release, but this caused
+  # issues since when vimPlugins was auto-updated, the override wasn't always updated, resulting in
+  # the index.js file being out of sync with the rest of the plugin. This override solves that
+  # problem by completely overriding the plugin with a release version.
```

Comment:
We don't need this much comment. One can use git blame to see a full explanation.
You could narrow it down to "only official releases contains the required index.js file"

Reply:
Changed.